test/8-data-benchmarks.js

```diff
@@ -12,7 +12,10 @@ describe('8 - does some benchmarks on api calls, data events and events', functi
 
   this.timeout(120000);
 
-  var maximumPings = 1000;
+  // var maximumPings = 1000;
+  var maximumPings = 900; // with local exchange methods calling directly instead of through happn this
```

Comment:
I think we should leave in the setImmediate on callback. Forces async behaviour. We might take lose some performance, but I think it will be safer.